v3-docs/docs/.vitepress/config.mts

```diff
@@ -276,8 +279,17 @@ export default defineConfig({
     logo: { dark: "/meteor-logo.png", light: "/meteor-blue.png" },
 
     search: {
-      provider: "local",
+      provider: 'algolia',
+      options: {
+        appId: '2RBX3PR26I',
+        apiKey: '7fcba92008b84946f04369df2afa1744',
```

Comment:
I believe this api key refers to [search api key]( https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key), [not the admin one](https://www.algolia.com/doc/guides/security/api-keys/#admin-api-key), right? So it is safe to be publicly available, even though it may expose and [have some risks in place](https://www.algolia.com/doc/guides/security/api-keys/#risks-of-api-key-exposure).

Don't we have a secret management tool in place?

Reply:
Yeah, it is the public one!

VItepress apiKry is also public https://github.com/vuejs/vitepress/blob/f65053c1c6d15a8a7867a5363484656a6ba766f1/docs/.vitepress/config/shared.ts#L57

We could but it is not necessary I think

Reply:
I agree with @Grubba27. I think there is no problem